src/containers/table-container-generator/table-container-generator.js

```diff
@@ -27,21 +32,23 @@ const TableContainerGenerator = ({ tableTitles, tableItems, pagination }) => {
           <TableBody id='table-body'>{tableItems}</TableBody>
         </Table>
       </TableContainer>
-      {pagination && <TablePaginator />}
+      {pagination && count > config.maxItemsPerPage && <TablePaginator />}
```

Comment:
can it make sense to descructure this?
const { maxItemsPerPage } = config;

Reply:
fixed